### ISSUE TITLE
REPL in webstorm

### ISSUE THREAD

**Issue by alexeygolev** [Atsky/haskell-idea-plugin#112]
Possibly my bad, but I can't seem to start REPL. Gives me "Error:Invalid SDK Home path set. Please set your SDK path correctly." However everything is seem to be set correctly. Unless I don't understand what SDK is.
Webstorm 10.0.4, MacOs X 10.10.2

**Comment by Atsky** [comment]
This is a problem with plugin. I primarily support the Intellij Idea, and some features not so good tested in other products.

**Comment by alexeygolev** [comment]
Ok, thank you

**Comment by perrytew** [comment]
I'm new to haskell, and I received the same error. 
Here are the steps I took to fix it:

Make sure you have created a haskell project.
**First Steps**
File -> Project Structure
Under Platform Settings, click on SDKs tab
Click the green "plus" sign to add an SDK.
Select GHC from the drop down choices.
Navigate to the proper directory.  For me, it was C:\Program Files\Haskell Platform\8.0.1
Click OK

**Second Steps**
File -> Project Structure
Under Project Settings, click on Project tab
In the Project SDK dropdown, select the SDK you just created in "First Steps" above.
Click OK or Apply

That should fix your problem.
![haskell_intellij_sdk_1](https://cloud.githubusercontent.com/assets/3085264/18074592/c0d2457c-6e3b-11e6-961d-25b7ebcfb6b4.png)

![haskell_intellij_sdk_2](https://cloud.githubusercontent.com/assets/3085264/18074593/c5947f30-6e3b-11e6-99d0-11bc1d9cc18d.png)

**Comment by koalixCZ** [comment]
perrytew, this is a solution for the Intellij IDEA (thanks for them), but not for WebStorm. I have set cabal, (ghc-mod, ghc-modi) but when I try to start Haskell REPL, got the same problem with the invalid SDK path.

**Comment by perrytew** [comment]
I have WebStorm as well.  I'll have a look.

**Comment by perrytew** [comment]
I don't think this plugin, in its current form, will work with WebStorm.  WebStorm has no ability to handle SDKs.  The external libraries for WebStorm are javascript only.

**Comment by koalixCZ** [comment]
Thank you for your help. I will try Idea. I'm starting with a Haskell and looking for some editor/IDE.

**Comment by perrytew** [comment]
The Intellij community edition is free, and should provide the exact same experience as the enterprise edition with respect to Haskell.

**Comment by greg-murray-volusion** [comment]
So JetBrains should filter out or mark plugins that are not supported in Webstorm for a better customer experience.  I could swear it worked a few years ago.  Wasted an hour+
